jaeger-core/src/main/java/com/uber/jaeger/reporters/protocols/JaegerThriftSpanConverter.java

```diff
@@ -48,7 +48,7 @@ private JaegerThriftSpanConverter() {}
             context.getTraceId(),
             0, // TraceIdHigh is currently not supported
             context.getSpanId(),
-            context.getParentId(),
+            span.getParentContext() == null ? 0 : span.getParentContext().getSpanId(),
```

Comment:
not sure I follow this one. The SpanContext class still has the parentID field, and it's going to be used by injectors, so why do we need to change this? And if we don't, then it looks like we don't need  `span.getParentContext` to be public